pallets/asset-manager/src/migrations.rs

```diff
@@ -65,22 +67,18 @@ impl<T: GetStorageVersion + Config + PalletInfoAccess> OnRuntimeUpgrade
     #[cfg(feature = "try-runtime")]
     fn pre_upgrade() -> Result<(), &'static str> {
         let storage_version = <T as GetStorageVersion>::on_chain_storage_version();
-
         if storage_version >= 1 {
```

Comment:
If it's bumped, change `1` to `2`.

Reply:
This migration is not actually executed, but i'll add these checks to the asset-id migration as well.